web-admin/src/features/bookmarks/HomeBookmarkButton.svelte

```diff
@@ -19,13 +19,12 @@
   export let project: string;
   export let exploreName: string;
   export let homeBookmark: BookmarkEntry | undefined;
+  export let defaultExploreUrl: string;
```

Comment:
Because our concept of a "default" Explore is so confusing (which default is it?), we need to be clear about where this state is coming from. How about `urlForExploreYAMLDefaultState`?

If our state system allowed for it, another option would be to pass the `exploreState` object into this component and call `.toURL()`
```
export let exploreStatefromExploreYAMLDefaults: ExploreState

$: homeBookmarkUrl = homeBookmark?.url ?? exploreStatefromExploreYAMLDefaults.toURL();
```

Reply:
`exploreState` is a plain object. If and when we move it a class we can add methods like these.